redis/Cargo.toml

```diff
@@ -15,3 +15,4 @@ redis = "0.16"
 
 [dev-dependencies]
 tokio = { version = "0.2", features = ["macros"] }
+futures-util = "0.3"
```

Comment:
this is just need for the new redis examples, let me know if there is a better way of doing this ?